source/agora/crypto/Hash.d

```diff
@@ -121,7 +121,7 @@ public Hash hashFull (T) (scope const auto ref T record)
 
 /// Ditto
 public void hashPart (T) (scope const auto ref T record, scope HashDg hasher)
-    /*pure*/ nothrow @nogc
+    /*pure*/ nothrow @nogc @trusted
```

Comment:
Is the `@trusted` here necessary? You already labeled `toVarInt` as `@trusted`.

Reply:
It seems it can be removed now.